### ISSUE TITLE
Unable to use HTML in cells or header

### ISSUE THREAD

**Issue by sasha42** [grid-js/gridjs-vue#40]
Whenever I try to use the [HTML in Header](https://gridjs.io/docs/examples/html-header-cells) feature of Grid.js, I hit errors with gridjs-vue. I believe that this is because I'm unable to import properly the `html` module from Grid.js when using gridjs-vue.

Here is how I'm importing it:
```import {Grid, html} from "gridjs-vue";```

And this is the error that I'm seeing in Vue (the table does not get rendered):
```
[Vue warn]: Unknown custom element: <grid> - did you register the component correctly? For recursive components, make sure to provide the "name" option.

found in

---> <Parse> at src/components/HelloWorld.vue
       <App> at src/App.vue
         <Root>
```

Am I doing something wrong?

**Comment by selfagency** [comment]
`gridjs-vue` does not presently export `html`. You would need to require/import `gridjs` separately.

I will be updating the component sometime this week to reflect new `gridjs` API changes.

**Comment by sasha42** [comment]
Awesome, thanks for your help! We tried to use gridjs separately, but there appears to be a memory leak somewhere which leads to huge RAM consumption (anything past 1mb of data was unusable).

Let me know if you need someone to beta test your changes! 😄

**Comment by selfagency** [comment]
This is resolved in the now-available v0.4.0 release. See the README for directions.
